v0.3/cruorg/components/searchbar/searchbar.css

```diff
@@ -0,0 +1,92 @@
+/* Searchbar styles */
```

Comment:
Is this supposed to be for https://github.com/adobe/aem-core-wcm-components/tree/master/content/src/content/jcr_root/apps/core/wcm/components/search/v1/search or something else?

Reply:
@wrandall22 I believe so. That's the only core search component I'm seeing. The HTML was taken from [here](https://www.aemcomponents.dev/content/core-components-examples/library/commerce/search.html).

Reply:
Hmm, I think the HTML there is not the same as what our core component has. From my local:

```
<div class="search cru-searchbar-dark aem-GridColumn aem-GridColumn--default--12">
<section class="cmp-search" role="search" data-cmp-min-length="3" data-cmp-results-size="10">
    <form class="cmp-search__form" data-cmp-hook-search="form" method="get" action="/content/cru/us/en/metadata/landing.searchresults.json/_jcr_content/content-parsys/search" autocomplete="off" _lpchecked="1">
        <div class="cmp-search__field">
            <i class="cmp-search__icon" data-cmp-hook-search="icon"></i>
            <span class="cmp-search__loading-indicator" data-cmp-hook-search="loadingIndicator"></span>
            <input class="cmp-search__input" data-cmp-hook-search="input" type="text" name="fulltext" placeholder="Search" role="combobox" aria-autocomplete="list" aria-haspopup="true" aria-invalid="false" aria-owns="cmp-search-results-0">
            <button class="cmp-search__clear" data-cmp-hook-search="clear" aria-hidden="true" style="display: none;">
                <i class="cmp-search__clear-icon"></i>
            </button>
        </div>
    </form>
    <div class="cmp-search__results" data-cmp-hook-search="results" role="listbox" aria-multiselectable="false" id="cmp-search-results-0"></div>
    
<script data-cmp-hook-search="itemTemplate" type="x-template">
    <a class="cmp-search__item" data-cmp-hook-search="item">
        <span class="cmp-search__item-title" data-cmp-hook-search="itemTitle"></span>
    </a>
</script>
</section></div>
```

Reply:
Well that's annoying. The component library with the markup I've been using said it was all from that Adobe WCM repo, but apparently the components under the "Commerce" section aren't. They're part of the [aem-core-cif-components](https://github.com/adobe/aem-core-cif-components/) repo. I'll get it fixed.

Reply:
@wrandall22 This has been updated. The original component is still there under `searchbar` and the new one is simply `search`.